pkg/webhooks/pod/mutating.go

```diff
@@ -47,17 +59,54 @@ func (m *MutatingAdmission) Default(ctx context.Context, obj runtime.Object) err
 		return err
 	}
 
-	if ls.Matches(labels.Set(pod.Labels)) {
-		if m.Config.QOSInitializer.InitContainerTemplate != nil {
-			pod.Spec.InitContainers = append(pod.Spec.InitContainers, *m.Config.QOSInitializer.InitContainerTemplate)
+	if !ls.Matches(labels.Set(pod.Labels)) {
+		klog.Infof("injection skipped: webhook is not interested in the pod")
```

Comment:
injection->Injection, and this is a debug log, I suggest to add a log level and do not enable by default, and if we have label config for the webhook, this line of code should not be called?

Reply:
In fact, I don't like this logic. But in order to be compatible with the previous design, I still leave the matching here.